schema.json

```diff
@@ -316,59 +301,158 @@
       "type": "array"
     },
     "meta": {
-      "description": "The schema version and any other tooling configuration lives here",
+      "description": "the schema version and any other tooling configuration",
       "properties": {
         "canonical": {
           "description": "URL (as per RFC 3986) to latest version of this document",
           "type": "string"
         },
         "lastModified": {
-          "description": "e.g. 2017-06-29T15:53:00 - [resume.json uses the ISO 8601 date standard]",
+          "description": "date-time of last modified",
           "format": "date-time",
           "type": "string"
         },
+        "uuid": {
+          "description": "uuid v4 of the resume",
```

Comment:
Description is ok. In general v4 isn't that explanatory to non IT people so we could have said:
"Unique and random ID identifying each resume"